src/main/java/org/datavyu/plugins/ffmpeg/MpvMediaPlayer.java

```diff
@@ -2,14 +2,16 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.datavyu.plugins.DatavyuMediaPlayer;
+import org.datavyu.plugins.MediaException;
 import org.datavyu.util.LibraryLoader;
 import java.net.URI;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-public abstract class MpvMediaPlayer extends FfmpegMediaPlayer {
+abstract class MpvMediaPlayer extends DatavyuMediaPlayer {
```

Comment:
Why is still abstract?

Reply:
Because the init method is not implemented in MpvMediaPlayer nor in DatavyuMediaPlayer class, this class to abstract to allow child classes to implement the init method.